bin/collator/src/parachain/service.rs

```diff
@@ -620,7 +620,6 @@ fn start_aura_consensus(
                     .map(|c| ValidationCode::from(c).hash())
             }
         },
-        sync_oracle: sync_oracle.clone(),
```

Comment:
Quick question: Why do we have to remove this from the Aura parameters?

Reply:
It was never used and was removed in this commit - [paritytech/polkadot-sdk@`70afa25` (#4097)](https://github.com/paritytech/polkadot-sdk/pull/4097/commits/70afa254161517352fbf0da5346b25cae21ca40e)